keyboards/reto60/readme.md

```diff
@@ -0,0 +1,25 @@
+# reto60
+
+This keyboard is part of a end of engineering degree project, it is a compact 60% format keyboard with a ISO-ES Layout.
+
+* Keyboard Maintainer: [A.Garcia](https://github.com/TactioEarth)
+* Hardware Supported: Custom made reto60 PCB (ATMega32U4 MCU)
+* Hardware Availability: not available
```

Comment:
It isnt available atm, in the future this may change. It is in order to be able to flash and compile to get the hex. archive and configure further on with the qmk configurator.